app/models/concerns/campaigns/budgetable.rb

```diff
@@ -42,13 +42,13 @@ def selling_price
 
     # Returns a Money indicating how much budget has been spent
     def total_consumed_budget
-      gross_revenue start_date, Date.current
+      @total_consumed_budget ||= gross_revenue start_date, Date.current
```

Comment:
I'd throw parens on this now that it's on the right hand side of an assignment.

Reply:
I'd be wary of memoizing this unless you saw a big performance win from adding it.